src/data/daily_providers_metrics.csv.sh

```diff
@@ -16,8 +16,8 @@ COPY (
 
     select
         date,
-        count(distinct provider_id)  filter (where spark_retrieval_success_rate > 0.90) as meet_retrieval_sli,
-        count(distinct provider_id)  filter (where sector_health_rate between 0.95 and 1.00) as meet_sector_health_sli,
+        count(distinct provider_id) filter (where spark_retrieval_success_rate > 0.90) as meet_retrieval_sli,
```

Comment:
```suggestion
        count(distinct provider_id) filter (where spark_retrieval_success_rate >= 0.90) as meet_retrieval_slo,
```

I'm not sure if I have the syntax right.  

Should this be `>=` given .9 is meeting?

I think it makes sense to say "I met an SLO" but not "I met an SLI" which is why I adjusted it.

Reply:
Makes sense!